src/Shared/LayoutRenderers/AspNetResponseCookieLayoutRenderer.cs

```diff
@@ -93,6 +105,92 @@ private HttpCookieCollection GetCookies(HttpResponseBase response)
+        {
+            var expandMultiValue = OutputFormat != AspNetRequestLayoutOutputFormat.Flat;
+            return HttpCookieCollectionValues.GetVerboseCookieValues(cookies, CookieNames, Exclude, expandMultiValue);
+        }
+
+        private void SerializeAllProperties(IEnumerable<HttpCookie> verboseCookieValues, StringBuilder builder, LogEventInfo logEvent)
+        {
+            switch (OutputFormat)
+            {
+                case AspNetRequestLayoutOutputFormat.Flat:
+                    SerializeAllPropertiesFlat(verboseCookieValues, builder, logEvent);
+                    break;
+                case AspNetRequestLayoutOutputFormat.JsonArray:
+                    SerializeAllPropertiesJson(verboseCookieValues, builder);
+                    break;
+            }
+        }
+
+        private void SerializeAllPropertiesJson(IEnumerable<HttpCookie> verboseCookieValues, StringBuilder builder)
+        {
+            var firstItem = true;
+
+            foreach (var cookie in verboseCookieValues)
+            {
+                if (firstItem)
+                {
+                    builder.Append('[');
+                }
+                else
+                {
+                    builder.Append(',');
+                }
+
+                builder.Append("{");
+
+                AppendJsonProperty(builder, "Name", cookie.Name);
+                builder.Append(",");
+                AppendJsonProperty(builder, "Value", cookie.Value);
+                builder.Append(",");
+                AppendJsonProperty(builder, "Domain", cookie.Domain);
+                builder.Append(",");
+                AppendJsonProperty(builder, "Path", cookie.Path);
+                builder.Append(",");
+                AppendJsonProperty(builder, "Expires", cookie.Expires.ToUniversalTime().ToString("u"));
+                builder.Append(",");
+                AppendJsonProperty(builder, "Secure", cookie.Secure.ToString());
+                builder.Append(",");
+                AppendJsonProperty(builder, "HttpOnly", cookie.HttpOnly.ToString());
+
+                builder.Append("}");
+
+                firstItem = false;
+            }
+
+            if (!firstItem)
+            {
+                builder.Append("]");
+            }
+        }
+
+        private void SerializeAllPropertiesFlat(IEnumerable<HttpCookie> verboseCookieValues, StringBuilder builder, LogEventInfo logEvent)
+        {
+            var objectSeparator = GetRenderedObjectSeparator(logEvent);
+            var itemSeparator = GetRenderedItemSeparator(logEvent);
+            var valueSeparator = GetRenderedValueSeparator(logEvent);
+
+            var firstItem = true;
+            foreach (var cookie in verboseCookieValues)
+            {
+                if (!firstItem)
+                {
+                    builder.Append(objectSeparator);
+                }
+                firstItem = false;
+                builder.Append($"Name{valueSeparator}{cookie.Name}{itemSeparator}");
```

Comment:
I know that string-interpolation is almost magical with some compilers, but until we are using those compilers, then please make a private helper method that appends to the StringBuilder without using string-interpolation (Similar to `AppendJsonProperty`)

Reply:
Added a helper method named AppendFlatProperty to MultiValueRendererBase